test/helper.js

```diff
@@ -94,9 +95,9 @@ exports.withServer = (mock, skipping) => {
     exports.load.cfg('taskcluster.accessToken', null);
     fakeauth.start(testClients, {rootUrl});
 
-    const api = await exports.load('api');
+    await exports.load('api');
```

Comment:
I *think* this can be omitted in this case

Reply:
I thought so too but tests started failing when I removed it, iirc. /me shrugs

Reply:
OK, that's not wholly surprising.. ordering is tricky in tests.